docker.local.cfg

```diff
@@ -0,0 +1,4 @@
+dspace.dir = /dspace
```

Comment:
As noted in today's meeting, let's move this to `[src]/src/main/docker/local.cfg`

Again, like in 6.x, let's add in a few comments at the top of the file to describe its usage (used by Docker / DockerFile)